packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.types.ts

```diff
@@ -88,6 +88,11 @@ export interface ISearchBoxProps extends React.InputHTMLAttributes<HTMLInputElem
    */
   clearButtonProps?: IButtonProps;
 
+  /**
+   * Optional name to use for the aria label on clear text button.
+   */
+  clearTextButtonAriaLabel?: string;
```

Comment:
can we follow the above naming convention? clearButtonProps, clearButtonAriaLabel

Reply:
definitely!